src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_COST = 40;
+
+  if (days < 3) {
+    return days * BASE_COST;
+  } else if (days < 7) {
+    return days * BASE_COST - 20;
```

Comment:
You should avoid using `else if` after an `if` that contains a `return` statement. The `else` keyword is not necessary because if the condition is met, the function will return and the subsequent code will not be executed.